01-supp-factors.Rmd

```diff
@@ -0,0 +1,189 @@
+---
+layout: page
+title: Programming with R
+subtitle: Understanding factors
+minutes: 20
+---
+
+```{r, include = FALSE}
+source('tools/chunk-options.R')
+```
+
+> ## Objectives {.objectives}
+>
+>   * Understand how to represent catagorical data in R
+>   * Know the difference between ordered and unordered factors
+>   * Be aware of some of the problems encountered when using factors
+
+This section is taken from the datacarpentry lessons git@github.com:datacarpentry/datacarpentry.git
```

Comment:
It's not identical to the Data Carpentry lesson. Shouldn't you instead say that this lesson was "inspired by", "modeled after", or something to that effect? Also, it would be more useful to provide a URL to the lesson and not the ssh address for cloning the repo.